discretize/View.py

```diff
@@ -737,7 +737,7 @@ def plotGrid(
     def plot_3d_slicer(self, v, xslice=None, yslice=None, zslice=None,
                        vType='CC', view='real', axis='xy', transparent=None,
                        clim=None, aspect='auto', grid=[2, 2, 1],
-                       pcolorOpts=None, fig=None):
+                       pcolorOpts=None, fig=None, showIt=False):
```

Comment:
I probably would prefer `showIt=True` as default. Otherwise I have to provide `showIt=True` every time I call the slicer and I want to see it right away? Or am I missing something?

Reply:
I think the original motivation for the `showIt` was really for testing (cc @rowanc1). We didn't have the `agg` backend set on travis, so showing the plots made a mess. At this point, I think we can leave it out, or at the very least set the default as `True`